spec/fake_app.rb

```diff
@@ -24,6 +24,6 @@ class Application < Rails::Application
   end
 end
 
-Rails.logger = Logger.new("/dev/null")
+Rails.logger = Logger.new(File::NULL)
```

Comment:
nice, didn't know about this :scribbles in notes:

Reply:
TIL too - thanks new `standard` check